README.md

```diff
@@ -23,7 +23,9 @@ To setup the PatternFly 4 development environment:
 
 - clone the project
 - run `npm install` from the project root
-- run `npm run dev:lite`
+- run `npm run build-patternfly`
```

Comment:
We still need `cli:setup` since that adds `pf.js` to your path so you can use `pf generate` commands

> - run `npm run cli:setup` (only needed if doing development)

Reply:
sorry missed it on commit